builtin/rebase.c

```diff
@@ -306,6 +324,12 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 			    oid_to_hex(&options.upstream->object.oid)),
 			    oid_to_hex(&options.orig_head));
 
+	strbuf_addf(&quiet_config, NULL, &options.quiet,
```

Comment:
That is an incorrect call of `strbuf_addf()` because the format parameter must not be `NULL`, it has to be a valid format specification (passing `NULL` will most likely cause a segmentation fault).

It is not immediately clear to me what you want to do here...

Reply:
Ah, I think I understand what is going on here. You wanted to translate this statement, right? https://github.com/prertik/git/blob/ae91dbb2556d7ea9f4e205fbf7847fa8579c64b4/TODO-rebase.sh#L358

But in that case, I would have strongly expected code like this, and much, much earlier at that (i.e. directly after the `parse_options()` call):

```c
if (opts->quiet)
    strbuf_addstr(&git_am_opt, " -q");
opts->git_am_opt = git_am_opt.buf;
```

... and I would have expected to see the definition `struct strbuf git_am_opt = STRBUF_INIT;` instead of `struct strbuf quiet_config = STRBUF_INIT;`.